tests/run/rooted_stringcontext.scala

```diff
@@ -0,0 +1,10 @@
+object StringContext {
+  val i = 42
+  val s = s"Answer: $i"
+}
+
+object Test {
+  def main(args: Array[String]): Unit = {
+    println(StringContext.s)
```

Comment:
I would remove the check file and simply do `assert(StringContext.s == "Answer: 42")`

Reply:
Done.